.github/workflows/build.yml

```diff
@@ -4,20 +4,14 @@ on: [push]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Scala
+        uses: olafurpg/setup-scala@v11
         with:
-          java-version: 1.8
-      - name: Download gradle
-        run: ./gradlew --version
-      - name: Run tests
-        run: ./gradlew test
-      - name: Run compile
-        run: ./gradlew compile
-      - name: Run build
-        run: ./gradlew build
\ No newline at end of file
+          java-version: openjdk@1.14

```

Comment:
is the project supposed to be compiled with Java 14?

Reply:
I think this should be ok (but correct me if I'm wrong) as the target JDK in SBT is 1.8.